lib/compilers/fake-for-test.ts

```diff
@@ -54,6 +54,7 @@ export class FakeCompiler implements ICompiler {
 
     initialise(mtime: Date, clientOptions: any, isPrediscovered: boolean) {
         throw new Error('Method not implemented.');
+        return null;
```

Comment:
O_o why is this needed, typescript please

Reply:
Yeah, this one might qualify as another ts bug :(